lib/brainstem/dsl/field.rb

```diff
@@ -51,6 +53,27 @@ def conditionals_match?(model, presenter_conditionals, helper_instance = Object.
           presenter_conditionals[conditional].matches?(model, helper_instance, conditional_cache)
         }
       end
+
+      private
+
+      def run_on_with_lookup(model, context, helper_instance)
```

Comment:
This seems like it may be identical to the `Association#run_on_with_lookup` - think it'd be worth added it as a mixin or something?